examples/SampleApp/src/screens/NewDirectMessagingScreen.tsx

```diff
@@ -222,24 +226,32 @@ export const NewDirectMessagingScreen: React.FC<NewDirectMessagingScreenProps> =
    *
    * 2. And then navigate to ChannelScreen
    */
-  const customSendMessage = async () => {
-    if (!currentChannel?.current) return;
+  const customSendMessage = async (
+    _: string,
+    message: StreamMessage,
+  ): Promise<SendMessageAPIResponse> => {
+    if (!currentChannel?.current) {
+      throw new Error('Missing current channel');
+    }
 
     if (isDraft.current) {
       currentChannel.current.initialized = false;
       await currentChannel.current.query({});
```

Comment:
should this be in the try block? As this all gets called in a try catch internally I suppose it isn't uncaught?

Reply:
I think it was in a try block already because it used to use the old sendMessage. I will check if its removable